source/Junkership.js

```diff
@@ -7,6 +7,10 @@ import Reference from "./Reference.js"
 import Projectile from "./Projectile.js"
 import Score from "./Score.js"
 import Sound from "./Sound.js"
+import {PowerUp, PeaShoota, TriShoota, FiveShoota, RapidFire, RapidSprayShot,
+    SprayShot, SuperSprayShot, CrazySprayShot, VertSprayShot,
+    VertShoota, BFG, Laser, PiercingLaser, SuperLaser, Mine,
+    SuperMine, PaintShot} from "./PowerUp.js"
```

Comment:
You don't actually need this or `this.WeaponList` anymore as I removed those dependencies a few PR's ago. :)